src/lib/FlightTasks/tasks/FlightTask.hpp

```diff
@@ -98,6 +98,14 @@ class FlightTask : public control::Block
 
 	static const vehicle_local_position_setpoint_s empty_setpoint;
 
+	/**
+	 * Call this whenever a parameter update notification is received (parameter_update uORB message)
+	 */
+	void handleParameterUpdate()
```

Comment:
I'm assuming this method is added such that a task can customize its checks/reactions to parameter changes right?

Reply:
The primary purpose to call the protected `updateParams` method. If needed, it can be used for additional checks as well, but it will need to be made `virtual`.